src/pk_encryption.rs

```diff
@@ -0,0 +1,115 @@
+use pyo3::{
+    pyclass, pymethods,
+    types::{PyBytes, PyType},
+    Bound, Py, Python,
+};
+use vodozemac::Curve25519PublicKey;
+
+use crate::PkEncryptionError;
+
+#[pyclass]
```

Comment:
Docs for this, PyO3 transforms the Rust docs into Python ones, I know I know, the rest of the bindings are missing docs as well, but that's something we need to work on as well.